client/src/helpers/nova/novaTimeUtils.ts

```diff
@@ -0,0 +1,58 @@
+import { ProtocolInfo } from "@iota/sdk-wasm-nova/web";
+
+export function unixTimestampToSlotIndexConverter(protocolInfo: ProtocolInfo): (unixTimestampSeconds: number) => number {
+    return (unixTimestampSeconds: number) => {
+        const genesisSlot = protocolInfo.parameters.genesisSlot;
+        const genesisUnixTimestamp = protocolInfo.parameters.genesisUnixTimestamp;
+        const slotDurationInSeconds = protocolInfo.parameters.slotDurationInSeconds;
+
+        const elapsedTime = unixTimestampSeconds - Number(genesisUnixTimestamp);
```

Comment:
wouldnt it be better keep using bigint to avoid JS number restrictions?

Reply:
I'm not sure why that timestamp is BigInt in the first place. 
But now that it is, it's honestly easier to convert that one place to number than everything else to BigInt and then back.
I don't think the JS number limit will be an issue here cause a timestamp will never come close to exceed the Number max value (in seconds).
To give a perspective:
this is JS max safe value `9007199254740991`
this is the date using a value 4 orders of magnitude smaller
![image](https://github.com/iotaledger/explorer/assets/6864498/fc96af04-f768-4d79-addb-9c0b45ebc81f)

Reply:
perfect thanks